Updated content in various nodes in NodeJS  -Why nodejs, Installing N… (#2019)
kamranahmedse/developer-roadmap: content/roadmaps/107-nodejs/content/101-nodejs-modules/100-commonjs-vs-esm.md

> # Commonjs vs esm
> 
> The CommonJS module system, on the other hand, is built into Node.js. Prior to the introduction of the ES module in Node.js, CommonJS was the standard for Node.js modules. As a result, there are plenty of Node.js libraries and modules written with CommonJS.
> 
> The ES module format was created to standardize the JavaScript module system. It has become the standard format for encapsulating JavaScript code for reuse.
> 
> <ResourceGroupTitle>Free Content</ResourceGroupTitle>
> <BadgeLink colorScheme='blue' badgeText='Read' href='https://nodejs.org/en/about/'>Official Website</BadgeLink>
> <BadgeLink colorScheme='yellow' badgeText='Read' href='https://reflectoring.io/nodejs-modules-imports/'>CommonJs VS ESM</BadgeLink> 

Reviewer: ```suggestion
# CommonJS vs ESM

The CommonJS module system, on the other hand, is built into Node.js. Before the introduction of the ES module in Node.js, CommonJS was the standard for Node.js modules. As a result, many Node.js libraries and modules are written with CommonJS.

The ES module format was created to standardize the JavaScript module system. It has become the standard format for encapsulating JavaScript code for reuse.

<ResourceGroupTitle>Free Content</ResourceGroupTitle>
<BadgeLink colorScheme='blue' badgeText='Read' href='https://nodejs.org/en/about/'>Official Website</BadgeLink>
<BadgeLink colorScheme='yellow' badgeText='Read' href='https://reflectoring.io/nodejs-modules-imports/'>CommonJs VS ESM</BadgeLink> 
```
